deploy.sh

```diff
@@ -45,54 +47,103 @@ printf "email: $INPUT_EMAIL\npassword: $INPUT_PASSWORD" >>~/.jexia/config.yml
 COMMAND=$(echo "jexia app deploy --project $INPUT_PROJECT_ID --app $INPUT_APP_ID --format shell")
 
 # If the user has set the `wait` input to true we add the `--wait` flag to the command
-if [ $INPUT_WAIT = true ]; then
+if [ "$INPUT_WAIT" = true ]; then
```

Comment:
`true` is also a string, so best to quote it (although we know it is always there and does not contain spaces, it is best to always quote strings)

```suggestion
if [ "$INPUT_WAIT" = "true" ]; then
```